src/brasil/gov/tiles/tiles/nitf.py

```diff
@@ -12,6 +13,20 @@ class INITFBasicTile(INITFTile):
     """A tile that shows general information about news articles.
     """
 
+    # FIXME: Há interesse de colocar esse campo no collective.cover, mas ainda
+    # sem previsão.
+    # https://github.com/plonegovbr/brasil.gov.tiles/pull/178#issuecomment-313092733
+    # A partir da versão 1.3b2, com a atualização do collective.nitf para 2.x e
+    # com o tile de NITF passando a herdar do tile de collective.nitf ao invés
+    # do basic em brasil.gov.tiles, esse atributo foi perdido e por isso ele
+    # foi adicionado aqui.
+    form.no_omit('image_description')
+    form.omitted(IDefaultConfigureForm, 'image_description')
+    image_description = schema.TextLine(
```

Comment:
I would recommend the use of `alt` as filed name to be consistent with the attribute; description should be something like "Replacement text for use when images are not available".